docs/core/entity/date.md

```diff
@@ -13,7 +13,7 @@ Properties should always only return information from memory and not do I/O (lik
 
 | Name | Type | Default | Description
 | ---- | ---- | ------- | -----------
-| native_value | date | **Required** | The value of the date.
+| native_value | <code>date &#124; None</code> | **Required** | The value of the date.
```

Comment:
Maybe clarify that it should be a `datetime.date` instance?

Reply:
```suggestion
| native_value | <code>datetime.date &#124; None</code> | **Required** | The value of the date.
```